### PR TITLE
[RFR] Display empty screen

### PR DESCRIPTION
**Pull request by xavierhans**
Display an empty board of the game pentago using makefile, docker and python.

[task trello](https://trello.com/c/Iep9mhzt/1-as-donald-i-can-launch-the-game-and-see-an-empty-board)

screen

- [x] Docker
- [x] Python
- [x] Screen
- [x] Unit test
Travis integration will be done in the next PR

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	docker build -t pentago-python .
+
+run: ## Run the game
+	$(DOCKER) python3 main.py
+
+test: ## Run the tests
+	$(DOCKER) python3 -m unittest discover
+
+lint: ## Run the linter
+	$(DOCKER) pycodestyle .
```
Comment (Kmaschta) [inline Makefile]: Please configure your IDE to add an empty line in the end of each files

File: src/main.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    print('   ------- ------- ------- ')
+    print('0 | o o o | o o o | o o o |')
+    print('1 | o o o | o o o | o o o |')
+    print('2 | o o o | o o o | o o o |')
+    print('   ------- ------- ------- ')
+    print('   ------- ------- ------- ')
+    print('3 | o o o | o o o | o o o |')
+    print('4 | o o o | o o o | o o o |')
+    print('5 | o o o | o o o | o o o |')
+    print('   ------- ------- ------- ')
```
Comment (fzaninotto) [inline src/main.py]: pentago is 6x6, why do you display a larger grid?

File: Makefile (hunk 1 above)
Comment (xavierhans) [reply Makefile]: Done

File: src/main.py [hunk 3]
```diff
@@ -1,42 +1,5 @@
...
-        print("The line number has to be a number between 0 and 5 included.")
-        return ask_line()
-    return line
-
-
-def ask_position():
-    column = ask_column()
-    line = ask_line()
-    print('{} {}'.format(column, line))
+from board import *
```
Comment (zyhou) [inline src/main.py]: Be careful with `*` import

File: README.md [hunk 4]
```diff
@@ -1,2 +1,2 @@
 # pentago-python
-Pentago game using python for console
\ No newline at end of file
+Pentago game using python for console
```
Comment (fzaninotto) [inline README.md]: missing installation, run, and test instructions.

**Pull request merged**